.github/workflows/ci.yml

```diff
@@ -209,7 +209,6 @@ jobs:
           --workspace
           ${{ steps.prepare.outputs.feature-flags }}
           --features expensive-tests
-          -- --ignored
```

Comment:
There are some serialization roundtrip tests in `zcash_primitives/src/transaction/tests.rs` that are marked both `#[ignore]` and `#[cfg(feature = "expensive-tests")]`. They pass and run relatively quickly, so I don't know what the intent was there. Perhaps both `#[ignore]` and `#[cfg(feature = "expensive-tests")]` should be removed from them.

Reply:
The commit that introduces this change removes the `ignore` from those tests. But I agree, I don't know why these are considered expensive.